Resources/config/services.yml

```diff
@@ -6,6 +6,9 @@ services:
     tags:
       - { name: twig.extension }
 
+  CrossKnowledge\DataTableBundle\DataTable\DataTableRegistry:
+    alias: crossknowledge_datatable.registry
+
   crossknowledge_datatable.registry:
     public: true
     class: CrossKnowledge\DataTableBundle\DataTable\DataTableRegistry
```

Comment:
According to https://symfony.com/doc/5.4/service_container/alias_private.html#aliasing
the service with aliasing, we set public: true

Can you explain why line10 has no public: true, and line13 should be public: false ?